feature/selector.py

```diff
@@ -663,6 +742,9 @@ def plot_importance(scores: pd.DataFrame,
     df = scores[columns].copy()
     df.fillna(0, inplace=True)
 
+    # Group by feature
```

Comment:
can we add, group by feature "for CV results" so that this operation is explicit about CV

Reply:
Updated in this [commit](https://github.com/fidelity/selective/pull/3/commits/65c53462b58b6aa13e3670d5e3fe6fef00abf5b4).